spec/models/pageflow/edit_lock_spec.rb

```diff
@@ -4,7 +4,8 @@ module Pageflow
   describe EditLock do
     describe 'active' do
       it 'excludes timed out EditLocks' do
-        lock = create(:edit_lock, updated_at: (EditLock::TIME_TO_LIVE + 1.minute).ago)
+        time_lock = Pageflow.config.edit_lock * EditLock::TIMER_TOLERANCE
+        lock = create(:edit_lock, updated_at: (Time.now - time_lock - 1.minute))
```

Comment:
The piece of logic `Pageflow.config.edit_lock * EditLock::TIMER_TOLERANCE` is now duplicated a lot. What do you think about extracting a static `EditLock.time_to_live` method?

Reply:
Yeah, that looks better.